pdc/apps/common/renderers.py

```diff
@@ -65,8 +65,10 @@
 """
 
 URL_SPEC_RE = re.compile(r'\$(?P<type>URL|LINK):(?P<details>[^$]+)\$')
-ORDERING_STRING = "\n * `ordering` is used to override the ordering of the results, the value could be: "
-ORDERING_STRING_NESTED = "The nested ordering is supported by using double underscore \_\_ ."
+ORDERING_STRING = """
+ * `ordering` (string) Comma separated list of fields for ordering results.
```

Comment:
Can we maybe add some comments that let user know if many ordering fields exist, the latter one will order on the previous ordering results? Maybe a nitpick :)

Reply:
Good English skill :)

Reply:
> Maybe it is worth to let the user knows the behaviour has been changed to a nice way.

AFAIK the sorting behavior with multiple fields is still handled by rest framework in the same way. I.e. all fields are used, if some items in `results` happen to be "equal" when sorted using the first field in `ordering`, the next field in `ordering` is used, etc.

> Good English skill :)

It took me too much time to come up with that short sentence. :)

Reply:
Ah. Ok. I was thinking /?ordering=A&ordering=B, then my mistake. But I am still thinking it is worth to add your sentence, maybe the user don't quite familiar with DRF, this is an explicit explanation.